### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by RomeoKs**
[DEMO LINK](https://RomeoKs.github.io/react_movies-list/)

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+
+.movies {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
+  gap: 20px;
+}
+
+.card-image img {
+  object-fit: contain;
+}
```
Comment (CharAnagn) [inline src/App.scss:29]: You forgot to move those two to their correspoding file

**Review by CharAnagn — APPROVED** [review]
GG 🔥

File: src/components/MovieCard/MovieCard.tsx [hunk 2]
```diff
@@ -1,7 +1,51 @@
...
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Prop = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Prop> = ({ movie }) => (
+  <div className="card" data-cy="Movie" key={movie.imdbId}>
```
Comment (naziamoff) [inline src/components/MovieCard/MovieCard.tsx:11]: you should move each attribute to the new line if you have more than 2 of them

File: src/components/MovieCard/MovieCard.tsx [hunk 3]
```diff
@@ -1,7 +1,51 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Prop = {
```
Comment (naziamoff) [inline src/components/MovieCard/MovieCard.tsx:6]: can have the name `Props` as well

**Review by naziamoff — APPROVED** [review]
🔥

File: src/components/MovieCard/MovieCard.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (RomeoKs) [reply src/components/MovieCard/MovieCard.tsx:11]: I'll have to look more deep into it.

Thank you for the review and suggestions!

[the remark this answers is not in the document]
Comment (RomeoKs) [reply src/components/MovieCard/MovieCard.tsx]: Solved it with the last commit.

Thank you Babis @CharAnagn!

**Pull request closed without merging**